tests/InputFiles/ExportCoordinates/Input2D.yaml

```diff
@@ -7,12 +7,29 @@
 #   ExportCoordinates2DVolume0.h5
 
 DomainCreator:
-  Disk:
-    InnerRadius: 1.
-    OuterRadius: 3.
-    InitialRefinement: 1
-    InitialGridPoints: [3, 3]
-    UseEquiangularMap: true
+    Rectangle:
+      LowerBound: [-0.5, -0.5]
+      UpperBound: [0.5, 0.5]
+      IsPeriodicIn: [false, false]
+      InitialRefinement: [2, 2]
+      InitialGridPoints: [5, 5]
```

Comment:
In the 2d and 3d input sfile you're still changing the default domain and the indentation, please revert (at least make sure the indentation is correct, but if you have no reason to change the default domain then just revert the diff)

Reply:
Sorry I missed the indent fix...I've got that in the squash I'm about to push. There is a reason to change the domain: Disk and Shell don't support TimeDependence yet, and I wanted the input files to include time dependence for the test.

Reply:
I changed the domain to one of the appropriate dimension that supports TimeDependence (Disk and Shell don't yet). However, domains like Disk and Shell still work fine as long as you don't try to specify TimeDependence: in the input file.